### ISSUE TITLE
Need Help : Siteconfig Maker for xml file store is local

### ISSUE THREAD

**Issue by dbmtech** [K-vanc/Tempest-EPG-Generator#38]
How to make Siteconfig.php for xml file stored in local.?
For example xml file attached here.
[aro.zip](https://github.com/K-vanc/Tempest-EPG-Generator/files/12506187/aro.zip)

**Comment by Toei79** [comment]
you need a local server?

**Comment by dbmtech** [comment]
No, xml files stored in my pc, I want to make siteconfig for it.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Toei79** [comment]
oh theres a wiki pdf. looks some complicated imo. or needs practice.

**Comment by dbmtech** [comment]
I tried but not succeed, that's why I attached a file for example here. It will be great if you can make siteconfig for it for example. thanks!

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Toei79** [comment]
im no the developer.  i tried with tempest on other sites but  looks complicated, its easy i bet when you able to handle the lenguague used.

**Comment by dbmtech** [comment]
Ok

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by K-vanc** [comment]
hi @dbmtech ;

I have already explained you how to use local files, shared 1 siteconfig that I used and even prepared your version of siteconfig on below issue for dummy channels;

https://github.com/K-vanc/Tempest-EPG-Generator/issues/20#issuecomment-1446875548

now, you want to grab from a ready epg xml source generated from another grabber? What is the point of this? you can just link that file into your player or if you want to grab it with Tempest, It is generated from astro.com.my which Tempest siteconfig is available in Malaysia folder.

Anyway, just to reply your question in order to close out the issue, you need to point url1 to local file(xml, http, txt, csv, etc) with " file:// + full/path/of/your/file.xml" command as;

Example for Linux
file://var/www/html/yourfile.xml

Example for Windows
file://C:\Users\dbmtech\Epg\yourfile.xml

and I am sharing that example siteconfig.php for your attached file for the last time, but if you want to make your own siteconfigs then you have to learn it by doing yourself, not by raising issue for each time.. There are plenty of users learned how to make siteconfigs and now some of them are able to make even their unlocked versions of my [ENC] siteconfigs.

[dbmtect.zip](https://github.com/K-vanc/Tempest-EPG-Generator/files/12509716/dbmtect.zip)

hi @Toei79 ;

It has to be a little complicated since it is not an easy thing to be the most advanced programmable epg generator 🥇 but basicly everything else just regex(regular expression) which part of almost all the modern programming languages

**Comment by dbmtech** [comment]
Ok

**Comment by Toei79** [comment]
yea thats what i think lol, i mean if you handle it you see how easy it is pure logic i bet , barely i know pascal from sometime ago .  but you need learn the tools for use it.
